### PR TITLE
Create Gym Status component

### PR DESCRIPTION
**Pull request by jyxhazcake**
-Made a component that displays the gym's status
- Added an extra 'isOpen' property that takes in a boolean value which shows the open/closed status respectively.

image

Gym Open status:
image

Gym Closed status (isOpen = false):
image

### CODE REVIEW

**Comment by jyxhazcake** [comment]
[gym open/closed status](https://trello.com/c/lExDhR39/3-gym-open-closed-status)

File: frontend/src/components/GymStatus.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+`
+const Ellipse = styled.div<{ isOpen: boolean }>`
+  background: ${(props) => (props.isOpen ? `rgba(88, 185, 148, 1)` : `rgba(243, 117, 98, 1)`)};
+  height: 16px;
+  width: 16px;
+  border-radius: 50%;
+`
```
Comment (hui444) [inline frontend/src/components/GymStatus.tsx:13]: consider adding newlines between each styled-component

**Review by hui444 — APPROVED** [review]
lgtm!

File: frontend/src/components/GymStatus.tsx (hunk 1 above)
Comment (jyxhazcake) [reply frontend/src/components/GymStatus.tsx:14]: updated 5217711

**Pull request merged**